src/test/java/com/example/produktapi/model/ProductTest.java

```diff
@@ -1,56 +1,63 @@
 package com.example.produktapi.model;
 
+import com.example.produktapi.service.ProductService;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
```

Comment:
Flera onödiga importer sker. Inget större problem (särskilt då framtida tester i denna fil troligtvis kommer använda vissa av dem) men vi måste säkerställa att vi ej har kvar några sådana vid leverans av projektet till Simon.

```suggestion
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;
```

**Tips till fortsättningen:** IntelliJ visar varningar när man ska commita viss kod. Det kan vara värt att ta en titt där innan man commitar koden.